proto/pbservice/ids.go

```diff
@@ -22,12 +22,14 @@ func (m *CheckServiceNode) UniqueID() string {
 	switch {
 	case m.Node != nil:
 		builder.WriteString(m.Node.Partition + "/")
+		builder.WriteString(m.Node.PeerName + "/")
```

Comment:
Are we cool with a PeerName being empty here and have a `//` in the UniqueID?

Reply:
We already do this for Partition and Namespace here, as the `ids.go` file is not split.

There's already a test for:
```
		{
			name: "without partition or namespace or peer name",
			csn: &CheckServiceNode{
				Node: &Node{
					Node: "the-node-name",
				},
				Service: &NodeService{
					ID: "the-service-id",
				},
			},
			expected: "//the-node-name//the-service-id",
		},
```